src/scss/variables.scss

```diff
@@ -1,881 +1,974 @@
+
+$grid-columns:                12 !default;
+$grid-gutter-width:           30px !default;
+
+// Components
 //
-//## Specify custom location and filename of the included Glyphicons icon font. Useful for those including Bootstrap via Bower.
+// Define common padding and border radius sizes and more.
+
+$line-height-lg:              1.5 !default;
+$line-height-sm:              1.5 !default;
+
+$border-width:                1px !default;
+$border-color:                $gray-300 !default;
+
+$border-radius:               .4rem !default;
+$border-radius-lg:            .48rem !default;
+$border-radius-sm:            .32rem !default;
+
+$box-shadow-sm:               0 .125rem .25rem rgba($black, .075) !default;
+$box-shadow:                  0 .5rem 1rem rgba($black, .15) !default;
+$box-shadow-lg:               0 1rem 3rem rgba($black, .175) !default;
 
-//** Load fonts from this directory.
+$component-active-color:      $white !default;
+$component-active-bg:         theme-color("primary") !default;
 
-// [converter] If $bootstrap-sass-asset-helper if used, provide path relative to the assets load path.
-// [converter] This is because some asset helpers, such as Sprockets, do not work with file-relative paths.
-// Starter theme: Note that this may need to be adjusted if you are installing into a sub-folder rather than the
-// root of a domain
-$icon-font-path: "/themes/starter/dist/fonts/" !default;
-// Font-awesome path (using CDN to avoid subfolder installation problems)
-$fa-font-path: "//netdna.bootstrapcdn.com/font-awesome/4.7.0/fonts" !default;
+$caret-width:                 .3em !default;
 
-//** File name for all font files.
-$icon-font-name:          "glyphicons-halflings-regular" !default;
-//** Element ID within SVG icon file.
-$icon-font-svg-id:        "glyphicons_halflingsregular" !default;
+$transition-base:             all .2s ease-in-out !default;
+$transition-fade:             opacity .15s linear !default;
+$transition-collapse:         height .35s ease !default;
 
+$caret-width:                 .3em !default;
 
-//== Components
+// Fonts
 //
-//## Define common padding and border radius sizes and more. Values based on 14px text and 1.428 line-height (~20px to start).
+// Font, line-height, and color for body text, headings, and more.
 
-$padding-base-vertical:     6px !default;
-$padding-base-horizontal:   12px !default;
+// stylelint-disable value-keyword-case
+$font-family-sans-serif:      -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
+$font-family-monospace:       SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace !default;
+$font-family-base:            $font-family-sans-serif !default;
+// stylelint-enable value-keyword-case
 
-$padding-large-vertical:    10px !default;
-$padding-large-horizontal:  16px !default;
+$font-size-base:              1.6rem !default;
```

Comment:
This feels very wrong. I found this with the following investigation:
- Buttons feel wrong - the padding is off
- The padding is default bootstrap styling, using `rem`
- Why does this look so different from bootstrap docs?
- rem is based on the parents font-size
- The default font-size is 1.6rem 😕 
- 1.6rem should make everything big - the browser default is like 16px or something
- Oh there's a `font-size: 62.5%` on `html` for some reason? That feels weird - comment that out
- Now everything is huge. Because `1.6rem` on body is 16px (browser default) times 1.6 = 25.6px
- Change `$font-size-base` back to `1rem` - everything looks fine now.

Reply:
>`rem` is based on the parents `font-size`

My understanding is that you've described `em` where as `rem` is the same, excepting that the 'r' stands for "root" and is essentially the document's font size (root element).

The rest are all valid points though - the only thing to keep in mind here is that there were design changes to go along with this upgrade, and @clarkepaul has stated he's not fussed if things are a _little_ off from what they used to be.

Do you have any screenshots to exemplify your points @ScopeyNZ ?